pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitScanErrorTest.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.dsm.kits;
+
+
+import org.broadinstitute.dsm.model.kit.ScanError;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class KitScanErrorTest {
+
+    @Test
+    public void testIsScanErrorOnlyBspParticipantId(){
+        String kit = "TEST_KIT_LABEL";
+        String error = null;
+        String shortId = "TEST_SHORT_ID";
+
+        ScanError notRealScanError = new ScanError(kit, error, shortId);
+        Assert.assertTrue(notRealScanError.isScanErrorOnlyBspParticipantId("TEST_SHORT_ID"));
```

Comment:
Nice!  Maybe break these assertions out into three different test methods and then name them with along the lines of `testShortIdIsNotErrror`, `testNullShortIdNotError`, etc.  That way we'll have finer feedback about the nature of what's broken.